evap/student/views.py

```diff
@@ -41,38 +41,21 @@ def index(request):
     return render(request, "student_index.html", template_data)
 
 
-def vote_preview(request, course, for_rendering_in_modal=False):
```

Comment:
What do you think about chanting the default value for preview or removing it? This would force all calls to explicitly state whether we want a preview rendering where no submission can be made or a no-preview rendering which can be submitted which could help identifying bugs.